couchdb/server/compiler.py

```diff
@@ -78,3 +96,48 @@ def helper():
         'parent': mod,
         'id': (idx is not None) and (idx + '/' + name) or name
     })
+
+
+def import_b64egg(b64str, egg_cache=None):
+    """Imports top level namespace from base64 encoded egg file.
+
+    For Python 2.4 `setuptools <http://pypi.python.org/pypi/setuptools>`_
+    package required.
+
+    :param b64str: Base64 encoded egg file.
+    :type b64str: str
+
+    :return: Egg top level namespace or None if egg import disabled.
+    :rtype: dict
+    """
+    if iter_modules is None:
+        raise ImportError('No tools available to work with eggs.'
+                          ' Probably, setuptools package could solve'
+                          ' this problem.')
```

Comment:
No, [that one](https://github.com/kxepal/couchdb-python/blob/viewserver/couchdb/server/compiler.py#L14-L31), but it seems you already dropped it. Worth to check for `zipimport` and `get_importer` as well.